src/Components/ClientSuspense.tsx

```diff
@@ -0,0 +1,24 @@
+import { useDidMount } from "@artsy/palette"
+import { FC, ReactNode, Suspense } from "react"
+
+interface ClientSuspenseProps {
+  fallback: NonNullable<ReactNode>
+  children: ReactNode
+}
+
+export const ClientSuspense: FC<ClientSuspenseProps> = ({
```

Comment:
👍 

So damon, what would be your guidance re `useClientQuery`? Lean into suspense? Or use whats best?

It still feels way nicer having a `loading` prop and not have to deal with vertical component boundaries. Curious about yr take

Reply:
Yeah, I like it and find it pretty intuitive. We just can't use it with SSR given our current framework.

Reply:
Cool 👍